cogs/servermanage/core.py

```diff
@@ -0,0 +1,432 @@
+import asyncio
+from datetime import date, datetime, timedelta
+import logging
+from os import remove
+from os.path import splitext, join as pathJoin
+from pathlib import Path
+
+import discord
+
+from redbot.core import Config, data_manager
+from redbot.core.commands.context import Context
+from redbot.core.utils import AsyncIter
+from redbot.core.utils.chat_formatting import error, pagify, warning
+from redbot.core.utils.menus import DEFAULT_CONTROLS, menu
+from redbot.core.bot import Red
+
+from .constants import *
+from .exceptions import InvalidAttachmentsError, InvalidFileError, InvalidImageError
+
+
+class Core:
+    # Class constructor
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.config = Config.get_conf(self, identifier=5842647, force_registration=True)
+        # Register default (empty) settings.
+        self.config.register_guild(**BASE_GUILD)
+
+        # Initialize logger, and save to cog folder.
+        self.dataFolder = data_manager.cog_data_path(cog_instance=self)
+        self.logger = logging.getLogger("red.luicogs.ServerManage")
+        if not self.logger.handlers:
+            logPath = pathJoin(self.dataFolder, "info.log")
+            handler = logging.FileHandler(filename=logPath, encoding="utf-8", mode="a")
+            handler.setFormatter(
+                logging.Formatter("%(asctime)s %(message)s", datefmt="[%d/%m/%Y %H:%M:%S]")
```

Comment:
Not directly part of this PR, but can you fix this to be YYYY-MM-DD?

Reply:
Sure.